.github/workflows/pr-checks.yml

```
@@ -0,0 +1,116 @@
+    permissions:
+      contents: read
+
+  # All checks must pass
+  pr-gate:
+    name: PR Gate (All Checks)
+    needs: [ci-python-311, ci-python-312, pr-validation, reuse]
+    runs-on: ubuntu-latest
+    if: always()
+    steps:
+      - name: Check all jobs passed
+        run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:13:44: Double quote to prevent globbing and word splitting

(shellcheck)